### ISSUE TITLE
Mednaffe fails to build under Windows

### ISSUE THREAD

**Issue by Skalt711** [AmatCoder/mednaffe#163]
It expects mednaffe.res to be an i386 file, even though I build under a 64-bit architecture.

I use a MinGW64 build environment in an MSYS2 shell.

Logs:
https://pastebin.com/qGk7rDgF
https://pastebin.com/q5T502iR

P.S.: Same happens in a UCRT64 environment.

**Comment by AmatCoder** [comment]
Yes, _mednaffe.res_ in repository is a i386 file because Mednaffe is only distributed as a 32-bit application under Windows.

You need recompile it if you build it to another architecture.

**Comment by biggestsonicfan** [comment]
Do the resources for the `mednaffe.res` file exist to create a compatible 64-bit version?

**Comment by AntumDeluge** [comment]
I was able to get around it by editing `src/Makefile` & removing the reference to "../share/win/mednaffe.res".

@biggestsonicfan `share/win/mednaffe.rc` should be the source. But I ran into problems trying to link to it after compiling with `windres`.

```
../share/win/mednaffe.res: file not recognized: file format not recognized
```
